test/extensions/filters/network/ext_authz/ext_authz_test.cc

```diff
@@ -44,7 +46,8 @@ class ExtAuthzFilterTest : public testing::Test {
           "envoy_grpc": { "cluster_name": "ext_authz_server" }
       },
       "failure_mode_allow": true,
-      "stat_prefix": "name"
+      "stat_prefix": "name",
+      "include_peer_certificate": true
```

Comment:
this doesnt seem to be related to this change? or am i missing something

Reply:
oh if this is about my comment about verifying this for the network ext authz filter, I meant using the header formatting here to pass this information, not a dedicated field

Reply:
Sorry, can you clarify by: “using header formatting here”. Hum, network filter has its own impl that doesn’t use header formatter.

Reply:
Per offline conversation, this is reverted.